src/index.d.ts

```diff
@@ -17,6 +17,9 @@ export function observer<P, TFunction extends React.ComponentClass<P | void>>(ta
 export function inject<P>(...stores: string[]): (<TFunction extends IReactComponent<P>>(target: TFunction) => (TFunction & IWrappedComponent<P>)); // decorator signature
 export function inject<T, P>(storesToProps : IStoresToProps<T, P>): (<TFunction extends IReactComponent<T | P>>(target: TFunction) => (TFunction & IWrappedComponent<T>)); // decorator
 
+export function onError(cb: () => void): void;
```

Comment:
this is a little inconsistent with most mobx api's, where event handler registration returns a disposer functions, instead of having a separate off...